packages/prisma/zod-utils.ts

```diff
@@ -328,6 +328,7 @@ export const teamMetadataSchema = z
     subscriptionItemId: z.string().nullable(),
     isOrganization: z.boolean().nullable(),
     isOrganizationVerified: z.boolean().nullable(),
+    isOrganizationConfigured: z.boolean().nullable(),
```

Comment:
Whoops, added a new property here @keithwillcode. I will surely revise it after today's Engineering Round Table